src/Modal/Modal.js

```diff
@@ -100,6 +103,32 @@ function Modal({ children, title, animationDuration, showClose, onClose, open, .
     }
   }, [open]);
 
+  useEffect(() => {
+    if (blur && blurNode.current) {
+      const isBackgroundBlurred = !!blurNode.current.style.filter;
+      if (isOpen !== isBackgroundBlurred) {
+        blurNode.current.style.filter = isOpen ? 'blur(1px)' : null;
+      }
+    }
+  }, [isOpen]);
```

Comment:
we'll want to add `blur` to deps because the prop _could_ change between renders. 
```suggestion
  }, [isOpen, blur]);
```